client/src/components/sidebar/index.tsx

```diff
@@ -123,7 +123,7 @@ export const Sidebar = (props: Props) => {
                   </div>
                   <div className="relative mt-12 flex-1 px-2 flex flex-col h-full">
                     <Profile.Primary>
-                      <Profile.Primary.Image />
+                      <Profile.Primary.Image src={profile_pic} />
```

Comment:
why don't we import Profile with absolute path in line 11?
e.g. `@/components/Profile`

Reply:
Looks good. I applied your feedback. Thanks!